content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -16,6 +16,9 @@ weight: 10
 ## Security
```

Comment:
Another new security feature added to Istio 1.4:
**Added** experimental \[secure webhook management\](docs/tasks/security/webhook/).

Reply:
Its already in the istioctl section

Reply:
How about removing the one in the istioctl section and added this new security feature here since it is a security feature and users can easily find it under security?

Reply:
There are istioctl tools for install, telemetry, etc that have always been in istioctl section, so I think it makes sense to retain that unless docs owners suggest moving